ansible/roles/ocp4-workload-openbanking-sandbox/tasks/3amp-deploy-full-api.yml

```diff
@@ -53,13 +58,18 @@
     url: "{{ obsandbox_api_crds_base_url + '/product-' + _api_resource_id + '.yml' }}"
     dest: "{{ _crd_template.path }}"
 
+# Using 'cat' because look('file') or lookup('template') fails on remote host
+- name: Read Product CRD tempfile '{{ _crd_template.path }}' into '_crd_content' variable
+  shell: cat "{{ _crd_template.path }}"
```

Comment:
Same here. slurp would be the Ansible native way of doing this and avoiding shell.

Reply:
Made the changes. See the commit below.